docs/references/release-checklist.md

```diff
@@ -30,6 +30,7 @@ Before we release a new version or beta version the following checklist should b
   * Ensure the documentation on the public storybook site matches the latest release.
   * Open the storybook site and ensure the console log that lists the npm package version number matches the released package version number.
 * ✅ Post about release on the [internal releases Teams channel](https://teams.microsoft.com/l/channel/19%3ae12aa149c0b44318b245ae8c30365880%40thread.skype/ACS%2520Deployment%2520Announcements?groupId=3e9c1fc3-39df-4486-a26a-456d80e80f82&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47).
+* ✅ Delete the release branch from github wihtout merging back in so to not leave it hanging.
```

Comment:
Can you also update line 15 above. It is wrong. We should never cherry-pick string translations to/from `main`. Instead we should directly sync translations into the release branch using the dedicated GH action.